Sources/ClientRuntime/Networking/Http/CRT/CRTClientEngine.swift

```diff
@@ -165,6 +165,49 @@ public class CRTClientEngine: HTTPClient {
                 do {
                     let stream = try connection.makeRequest(requestOptions: requestOptions)
                     try stream.activate()
+                    if request.isChunked {
```

Comment:
This block of code now incorporates the code that used to be in the static method for chunking a body.